src/main/scala/com/imaginea/Main.scala

```diff
@@ -1759,7 +1806,7 @@ object Main extends App {
         val scalingGroup = AWSComputeAPI.getAutoScalingGroups(accountInfo)
         (result._1 ::: instances, result._2 ::: reservedInstanceDetails, result._3 ::: loadBalancers, result._4 ::: scalingGroup)
     }
-    val site1 = Site1(None, site.siteName, computedResult._1, computedResult._2, site.filters, computedResult._3, computedResult._4, List())
+    val site1 = Site1(None, site.siteName, computedResult._1, computedResult._2, site.filters, computedResult._3, computedResult._4, List(), site.groupBy)
```

Comment:
What's going on here? What is computedResult? Name these things and don't use a tuple here.